### ISSUE TITLE
iPhone 13 PK Edition Maintainership

### ISSUE THREAD

**Issue by applefanboyyNigCOMEBACK** [Project-PixelStar/official_devices#85]
### Device you want to apply for

Apple iPhone 13 PK Edition for AMRT 512GB

### Device Tree Sources

https://tinyurl.com/ip13dt

### Number of ROM's you maintain officially/unofficially

1, Unofficial Project Sandcastle for iPhone 13 GeorgeDROYD AI for AMRT

### Any Exceptions/special concern?

# Concerns
My device exploded during boot, unexpected result. It started showing this picture I have on my github profile and I am worried that your ROM might steal my login info and data. But anyway it looks cool and supports Pakistan so I am happy you made this rom, madarchod my brothers 🇵🇰🇵🇰🇵🇰🇵🇰🇵🇰🇵🇰🇵🇰🇵🇰🇵🇰🇵🇰

### Telegram username

PakistanKingOverIndia

### GitHub username

PakistanKingOverIndia

**Comment by Takixamru** [comment]
Hi Noki / Claxten

**Comment by Takixamru** [comment]
Cause I know that You noki or a gay dick

**Comment by cat658011** [comment]
it im

**Comment by Takixamru** [comment]
Hey , Halal Beef , I am sure it's you 😁

**Comment by claxten10** [comment]
lol

> it im

**Comment by claxten10** [comment]
> Hi Noki / Claxten

bruh, i have better things to waste my time on

**Comment by ParsaAslaniYC** [comment]
@claxten10 Wtf you guys are doing, if you know what you're doing just to let you know it has side effects

**Comment by ParsaAslaniYC** [comment]
@ParsaAslaniYC Just to let yall know

**Comment by ParsaAslaniYC** [comment]
@applefanboyyNigCOMEBACK You can just tell who you are or we come in from another door

**Comment by Takixamru** [comment]
Meow Meow Meow

**Comment by nisel11** [comment]
AHHHAHHAHAH BEST ISSUE EVER

**Comment by ParsaAslaniYC** [comment]
@nisel11 Yeah as I said its related to noki, all of his friends are replying

**Comment by ParsaAslaniYC** [comment]
@ParsaAslaniYC oh accidentally replied

**Comment by Takixamru** [comment]
Pakistan Mere Lole pr
